### ISSUE TITLE
[주제 2] test set 구조 및 제출 관련하여 질문드립니다.

### ISSUE THREAD

**Issue by Jwoo5** [DatathonInfo/H.D.A.I.2021#69]
1. 데이터가 부정맥인지 normal인지에 대한 레이블을 pre-process할 때 각 데이터 샘플이 위치한 디렉토리(arrhythmia / normal)를 기준으로 처리했는데요, test set도 train, validation set처럼 디렉토리 구조가 test/arrhythmia/sample.xml, test/normal/sample.xml 이런 식으로 구성되어있는지 여쭤봅니다.
2. 결과물 제출할 때 결과 요약지와 함께 학습코드, 모델 weight, 환경, 모델설명을 제출하라고 되어있는데요, 결과요약지 양식에 있는 모델 설명 외에 따로 모델설명을 위한 파일을 첨부해야 하나요? 또, 실험 환경은 어느 정도로 상세하게 기술하면 될까요?
3. 코드 실행가이드에 test뿐만 아니라 train을 어떻게 돌리는 지도 설명을 해야 하나요?
4. 마지막으로, AUC 출력값이 보이도록 찍은 스크린샷을 첨부하라고 되어 있는데, AUC 출력값이라는 게 validation set에 대한 auc curve를 plot해서 첨부하라는 의미인가요?

**Comment by DatathonInfo** [comment]
안녕하세요 참가자님,

1. Test Set의 구조는 제공해드린 데이터셋중 Validation Set과 동일하게 구성되어 있습니다.

2. 제출하실 환경은 모델이 구동되는 최소한의 환경을 명확하게 기재해주시기 바랍니다.
환경 미기재, 오기재로 인한 작동불능시 평가가 불가능할 수 있습니다.

3. 학습코드와 함께 코드 구동 방식도 제출 부탁드리겠습니다.

4. Validation Set의 AUROC Score를 출력 후 스크린샷 캡쳐해 주시면 됩니다.

감사합니다.
H.D.A.I.2021 운영팀 드림
